iina/InfoDictionary.swift

```diff
@@ -83,7 +87,6 @@ struct InfoDictionary {
   /// Enum corresponding to the build configurations in IINA's Xcode project.
   enum BuildType: String, CustomStringConvertible {
     case beta = "Beta"
-    case debug = "Debug"
     case nightly = "Nightly"
```

Comment:
This change makes the above comment incorrect. Xcode has a Debug build configuration, so you would expect it to be here. In other words this is no longer a direct mapping to Xcode configurations as the comment would have you believe. This is confusing and definitely needs a comment to explain the thinking. Possibly whatever we are trying to achieve is being done at too low of a level. In other words this low level code should do the simple thing and map to the Xcode configurations. That is easy and straight forward for people to understand. Higher level code can then treat the debug configuration differently, like mapping it to nightly.